backend/battle/templates/battle/pokemon_form.html

```diff
@@ -17,33 +17,57 @@
                 {% for hidden in form.hidden_fields %}
                     {{ hidden }}
                 {% endfor %}
-                <p>
-                    <label for="id_pokemon_1">Pokemon 1:</label>
-                    <input id="pokemon_1" name="pokemon_1" type="text" list="pokemons">
-                    <datalist id="pokemons" name="pokemon_1">
-                        {% for results in pokemons %}
-                            <option value="{{results.name}}">
-                        {% endfor %}
-                    </datalist>
-                </p>
-                <p>
-                    <label for="id_pokemon_2">Pokemon 2:</label>
-                    <input id="pokemon_2" name="pokemon_2" type="text" list="pokemons">
-                    <datalist id="pokemons" name="pokemon_2">
-                        {% for results in pokemons %}
-                            <option value="{{results.name}}">
-                        {% endfor %}
-                    </datalist>
-                </p>
-                <p>
-                    <label for="id_pokemon_3">Pokemon 3:</label>
-                    <input id="pokemon_3" name="pokemon_3" type="text" list="pokemons">
-                    <datalist id="pokemons" name="pokemon_3">
-                        {% for results in pokemons %}
-                            <option value="{{results.name}}">
-                        {% endfor %}
-                    </datalist>
-                </p>
+                <div class="form_pokemon_position">
+                    <p>
+                        <label for="id_pokemon_1">Pokemon 1:</label>
+                        <input id="pokemon_1" name="pokemon_1" type="text" list="pokemons">
```

Comment:
No form errors for this input here.

Reply:
I defined validation error for pokemons typed wrong way and file empty. So, I don't understand, what kind of form error?

Reply:
I mean printing the form errors like `{{ form.errors.pokemon_1 }}`

Reply:
Alright, thanks :)